app/src/main/java/me/hufman/androidautoidrive/carapp/music/views/BrowseView.kt

```diff
@@ -32,17 +34,20 @@ class BrowseView(val states: List<RHMIState>, val musicController: MusicControll
 	lateinit var inputState: RHMIState
 	lateinit var pageController: BrowsePageController
 	var lastApp: MusicAppInfo? = null
+	var currentPage: BrowsePageView? = null
 
 
-	fun initWidgets(playbackView: PlaybackView, inputState: RHMIState) {
+	fun initWidgets(playbackView: PlaybackView, inputState: RHMIState, musicApp: MusicApp) {
```

Comment:
The `initWidgets` function is used to link up a bunch of states together after they have all be constructed. The MusicApp could instead be passed to the BrowseView's constructor

Reply:
Ah that would be a much better spot for it, reverted the changes to the initWidgets() method and added the param to the BrowseView constructor